app/src/main/java/com/snatik/matches/fragments/DifficultySelectFragment.java

```diff
@@ -50,7 +56,57 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
 		animate(difficulty1, difficulty2, difficulty3, difficulty4, difficulty5, difficulty6);
 
+		Typeface type = Typeface.createFromAsset(Shared.context.getAssets(), "fonts/grobold.ttf");
+
+
+        TextView text1 = (TextView) view.findViewById(R.id.time_difficulty_1);
+		text1.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text1.setTypeface(type);
+        text1.setText(getBestTimeForStage(theme.id,1));
+
+        TextView text2 = (TextView) view.findViewById(R.id.time_difficulty_2);
+		text2.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text2.setTypeface(type);
+        text2.setText(getBestTimeForStage(theme.id,2));
+
+        TextView text3 = (TextView) view.findViewById(R.id.time_difficulty_3);
+		text3.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text3.setTypeface(type);
+        text3.setText(getBestTimeForStage(theme.id,3));
+
+        TextView text4 = (TextView) view.findViewById(R.id.time_difficulty_4);
+		text4.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text4.setTypeface(type);
+        text4.setText(getBestTimeForStage(theme.id,4));
+
+        TextView text5 = (TextView) view.findViewById(R.id.time_difficulty_5);
+		text5.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text5.setTypeface(type);
+        text5.setText(getBestTimeForStage(theme.id,5));
+
+        TextView text6 = (TextView) view.findViewById(R.id.time_difficulty_6);
+		text6.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL);
+		text6.setTypeface(type);
+        text6.setText(getBestTimeForStage(theme.id,6));
```

Comment:
format the code

Reply:
Sorry for repeatedly not formatting the way you want, it's just that when you have a certain style of coding its kinda hard to spot the changes you want. Regarding DifficultySelectFragment, as i showed you i do not know why the formatting looks wrong here, but in my device it appears correctly : 

![image](https://user-images.githubusercontent.com/15691073/30432789-51ac7404-996b-11e7-8946-c368a9de175d.png)

Reply:
Can you check if you have tabs or spaces before each line?

Reply:
The indentation is 4 spaces (1 tab)